### PR TITLE
Update 'postList.html' to use the base template and to contain photos

### LINKED ISSUE #94: update postList.html to use the base template

### LINKED ISSUE #101: Update 'postList' page to contain photos

### PR DESCRIPTION
**Pull request by Shoval-Ovadia**
**Updated 'postList.html' to use the base template** - 'Post/templates/Post/postList.html' modified.
Now this page will be in the agreed format.

**Update HTML page to contain photos:**
* Update 'localhost:8000/postList' so a post in the list will contain a photo -  'Post/templates/Post/postList.html' modified.
* Update 'test data' so the URL will be a valid one - 'Post/migrations/0002_test_data.py' modified.

closes #94
closes #101

### CODE REVIEW

File: Post/templates/Post/postList.html [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
 		<p>Place name: {{post.nameOfLocation}}</p>
 		<p>URL: {{post.photoURL}}</p>
 		<p>Description: {{post.Description}} </p>
 
 	{% endfor %}	
-</body>
-</html>
+
+
+{% endblock content %}
```
Comment (Yarboa) [inline Post/templates/Post/postList.html:13]: No new line?

**Comment by maayanhd** [comment]
I think it's too small of a change, I suggest changing it to consist of images based on URL as it suppose to be and it will help for the post view feature and even for the homepage perhaps, or at least make the changes that connect this page to the commenting page with a button or something else, as long as you want to change this particular file.

File: Post/templates/Post/postList.html (hunk 1 above)
Comment (Shoval-Ovadia) [reply Post/templates/Post/postList.html]: Fixed it.

**Comment by Shoval-Ovadia** [comment]
> I think it's too small of a change, I suggest changing it to consist of images based on URL as it suppose to be and it will help for the view post feature and even for the homepage perhaps, or at least make the changes for connecting this page to the commenting page with a button or something else, as long as you want to change this particular file.

Thanks for the suggestions, I will work on that.

**Review by Yarboa — APPROVED** [review]

**Pull request merged**